kiwi/container/__init__.py

```diff
@@ -30,9 +30,9 @@ class ContainerImage(object):
     :param string name: container system name
     :param string root_dir: root directory path name
     """
-    def __new__(self, name, root_dir):
+    def __new__(self, name, root_dir, custom_args=None):
```

Comment:
This was intentional as long as I was not 100% sure which parameters we need for umoci and skopeo. But I will adapt it. Thanks